.gitignore

```diff
@@ -9,3 +9,6 @@ classes
 # mac os x
 .DS_Store
 
+# indelllij
+.idea
+opel.iml
```

Comment:
*.iml if any (people could name their project different)

BTW I think, you should have this line in your machine gitignore
I totally agree with @pbetkier http://pbetkier.blogspot.com/2015/02/keep-your-gitignore-clean-as-well.html